xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -157,3 +172,31 @@ CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::Load(CVideoInfoTag& tag, bool)
 
   return CInfoScanner::NO_NFO;
 }
+
+CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::LoadMP4(CVideoInfoTag& tag)
+{
+  AVDictionaryEntry* avtag=nullptr;
+  while ((avtag = av_dict_get(m_fctx->metadata, "", avtag, AV_DICT_IGNORE_SUFFIX)))
+  {
+    if (strcmp(avtag->key, "title") == 0)
+      tag.SetTitle(avtag->value);
+    else if (strcmp(avtag->key, "composer") == 0)
+    {
+      std::vector<std::string> writers = StringUtils::Split(avtag->value, " / ");
+      tag.SetWritingCredits(writers);
```

Comment:
Could be `tag.SetWritingCredits(StringUtils::Split(avtag->value, " / "))`

Reply:
done.